test/framework/namespace.go

```diff
@@ -39,27 +39,27 @@ func CreateNamespace(kubeClient kubernetes.Interface, name string) (*v1.Namespac
 	return namespace, nil
 }
 
-func (ctx *TestCtx) CreateNamespace(t *testing.T, kubeClient kubernetes.Interface) string {
+func (f *Framework) CreateNamespace(t *testing.T, ctx *TestCtx) string {
```

Comment:
we could collapse the `CreateNamespace()` function above in this function. AFAICT it's only used here so there's little point in having it separate.